tests/Sniffs/Operators/NegationOperatorSpacingSniffTest.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Operators;
+
+use SlevomatCodingStandard\Sniffs\TestCase;
+
+class NegationOperatorSpacingSniffTest extends TestCase
+{
+
+	public function testNoErrors(): void
+	{
+		self::assertNoSniffErrorInFile(self::checkFile(__DIR__ . '/data/NegationOperatorSpacingSniffNoErrors.php'));
+	}
+
+	public function testErrors(): void
+	{
+		$file = self::checkFile(
+			__DIR__ . '/data/NegationOperatorSpacingSniffRequireSpaceErrors.php',
+			['requireSpace' => true]
```

Comment:
Please add both tests (errors and no errors) also for `false` (or `0` after the change).

Reply:
There is no-errors for withoutSpace variant, and with errors for with space variant ... isn't that enough?

Reply:
Is not. Now you don't test fixer for `requireSpace=false`

Reply:
Yeah makes sense.